sycl/test-e2e/lit.cfg.py

```diff
@@ -385,7 +386,8 @@
 
 if config.run_launcher:
     config.substitutions.append(('%e2e_tests_root', config.test_source_root))
-    config.recursiveExpansionLimit = 10
+
+config.recursiveExpansionLimit = 10
```

Comment:
Now that it isn't tied to the `run_launcher` we can move it somewhere else, I think.

Reply:
Yes, good idea. I moved it to the beginning, and copied an explanatory comment from compiler-rt for good measure.